package.json

```diff
@@ -24,6 +24,7 @@
     "mustache": "^4.2.0",
     "qrcode": "^1.5.3",
     "randomcolor": "^0.6.2",
+    "uuid": "^10.0.0",
```

Comment:
All major browsers have had support for `crypto.randomUUID()` for a long while, so we should use that instead of delivering this lib to the client.

Reply:
Wow, that's awesome, didn't know about that!

Reply:
Addressed in 32ff24e